electron/src/menu/system.ts

```diff
@@ -381,16 +381,13 @@ const processMenu = (template: Iterable<MenuItemConstructorOptions>, language: l
 
 const showRestartMessageBox = async () => {
   const {response} = await dialog.showMessageBox({
-    buttons: [
-      locale.getText('restartLater'),
-      EnvironmentUtil.platform.IS_MAC_OS ? locale.getText('menuQuit') : locale.getText('restartNow'),
-    ],
+    buttons: [locale.getText('restartLater'), locale.getText('restartNow')],
     message: locale.getText('restartLocale'),
     title: locale.getText('restartNeeded'),
     type: 'info',
   });
   if (response === 1) {
-    await (EnvironmentUtil.platform.IS_MAC_OS ? lifecycle.quit() : lifecycle.relaunch());
+    await lifecycle.relaunch();
```

Comment:
is this change on purpose? It seems unrelated and sounds like a regression

Reply:
Related to my other macOS change (see my first comment). The issue you highlighted previously with restart was not visible in my testing. I removed it in two locations.